models/hashtag.go

```diff
@@ -20,6 +21,42 @@ type Hashtag struct {
 	UpdatedUnix  int64
 }
 
+// Gets a summary of the hashtags for repositories with the specified prefix.
```

Comment:
As per Go convention, docs for a function should start with the function name (see [here](https://blog.golang.org/godoc-documenting-go-code))